src/reporting/ReputationToken.sol

```diff
@@ -11,19 +11,17 @@ import 'ROOT/reporting/Interfaces.sol';
 contract ReputationToken is DelegationTarget, Typed, Initializable, StandardToken, IReputationToken {
     using SafeMath for uint256;
 
-    string public name;
-    string public symbol;
-    uint256 public decimals;
+    // Delegated constracts cannot use string values, so we use bytes32 values here
+    bytes32 constant public name = "Reputation";
+    bytes32 constant public symbol = "REP";
```

Comment:
The ERC20 interface for these is `function name() constant returns (string)` and `function symbol() constant returns (string)`.  I'm not sure what to do about this, since we can't return a string from a delegated contract, and this doesn't match the ERC20 interface.